CefSharp.Core/Internals/CefTaskScheduler.h

```diff
@@ -42,7 +43,11 @@ namespace CefSharp
                 }
                 catch (Exception^)
                 {
-
+                    // This should never ever happen.
```

Comment:
But it would bubble into the CEF logic and not into .Net. (Since it's on the CEF thread) Otherwise I would have just removed it as you suggest.